command/assets/emoji.go

```diff
@@ -0,0 +1,21 @@
+package assets
+
+import (
+	"math/rand"
+	"time"
+
+	"github.com/bwmarrin/discordgo"
+)
+
+func RandomSelectEmoji(emojis []*discordgo.Emoji) string {
+	if len(emojis) < 1 {
```

Comment:
```suggestion
	if len(emojis) == 0 {
```
のほうがパッと見て意味がわかり易くない？
これは好みの問題な気がするので、もし < 1 のほうが私はいい！って場合は無視してもらって結構です。